package.json

```diff
@@ -30,6 +30,7 @@
     "react-table": "^7.0.0-beta.15",
     "react-use": "^13.9.0",
     "spectra-data": "^3.4.11",
+    "styled-components": "4.4.1",
```

Comment:
we already depend on the `emotion` library for styling, please use it. It has a similar API to create styled components: https://emotion.sh/docs/styled